### ISSUE TITLE
RFE: consider using pungi to do most all of the backend mash work

### ISSUE THREAD

**Issue by ralphbean** [fedora-infra/bodhi#1219]
It could be interesting.

- We could reduce the number of releng tools we have to worry about.
- Pungi can mash repos.
- Pungi can create ostree artifacts.
- Pungi produces metadata about *what it did*, which would be a handy thing to compare between runs.
- We could compare updates and ga releases in an *apples to apples* kind of way.

Ping releng members to see if they can provide some more motivation, details, or gotchas.

**Comment by bowlofeggs** [comment]
@jeremycline and @lubomir and I talked about this a bit today. We came to the conclusion that a good first step would be to have Bodhi use Pungi to do the OSTree composes. The immediate benefit to Fedora users will be that we would be able to release multi-arch atomic trees which we cannot do today. It will also allow us to gain some familiarity with using Pungi so that we are more poised to use it later to also build Yum repos.
